packages/@react-spectrum/list/src/listview.css

```diff
@@ -191,5 +191,7 @@
   justify-content: center;
   width: 100%;
   height: 100%;
+  .loadingMore {
+    padding-top: var(--spectrum-global-dimension-size-50);
+  }
```

Comment:
Padding is guess work on my part, borrowed the sizing from ListBoxBase. Didn't apply it in the layout because the ListView rows don't have padding in between